app/uk/gov/hmrc/apiplatformevents/models/common/models.scala

```diff
@@ -33,13 +33,21 @@ object ActorType extends Enum[ActorType] with PlayJsonEnum[ActorType] {
   case object UNKNOWN extends ActorType
 }
 
-case class Actor(id: String, actorType: ActorType)
+case class OldActor(id: String, actorType: ActorType)
+
+sealed trait Actor
+
+case class GatekeeperUserActor(user: String) extends Actor
+//case class CollaboratorActor(email: String) extends Actor
```

Comment:
Comments

Reply:
I think we said we should do this? Leave them commented out because not currently in use but have them in there to show how we intend it to be used

Reply:
Ah - my bad.